examples/example_cartesian_path.py

```diff
@@ -81,14 +82,21 @@
 plt.title("Joint Position Trajectories")
 for idx in range(q_vec.shape[0]):
     plt.plot(t_vec, q_vec[idx, :])
+
+line = plt.axvline(x=0, color="b")
```

Comment:
maybe initialize `curr_time` first and then you can do

```suggestion
line = plt.axvline(x=curr_time, color="b")
```